frontend/src/app/features/boards/new-board-modal/new-board-modal.component.ts

```diff
@@ -111,7 +132,12 @@ export class NewBoardModalComponent extends OpModalComponent {
     this.create({ type: 'free' });
   }
 
-  private createAction(attribute:string) {
+  private createAction(attribute:string):void {
+    if (this.eeShowBanners) {
+      this.toastService.addError(this.text.teaser_text);
```

Comment:
The error message should be different here I think. If I manage to click on the button (for whatever reason) without an EE token, the message is as following:
<img width="575" alt="Bildschirmfoto 2022-03-23 um 08 04 06" src="https://user-images.githubusercontent.com/7457313/159642501-75d5c226-5ab2-4aa0-b43d-9fff5a8d8b5b.png">